lib/godaddy-style.js

```diff
@@ -55,6 +53,18 @@ module.exports = function (gulp, options) {
       .pipe(jshintReporter);
   });
 
+  gulp.task('eslint-junit', 'Lint the js files with eslint and output junit results to file', function () {
+    var output = process.env.ESLINT_FILE || './build/lint/eslint-results.xml';
```

Comment:
can you change the default to ./build/test/...? I decided to only use a single directory for results.